sdk/communication/azure-communication-messages/src/main/java/com/azure/communication/messages/models/channels/WhatsAppMessageTemplateItem.java

```diff
@@ -118,10 +117,27 @@ public static WhatsAppMessageTemplateItem fromJson(JsonReader jsonReader) throws
             }
             WhatsAppMessageTemplateItem deserializedWhatsAppMessageTemplateItem
                 = new WhatsAppMessageTemplateItem(language, status);
-            MessageTemplateItemAccessHelper.setName(deserializedWhatsAppMessageTemplateItem, name);
+            deserializedWhatsAppMessageTemplateItem.name = name;
             deserializedWhatsAppMessageTemplateItem.kind = kind;
             deserializedWhatsAppMessageTemplateItem.content = content;
             return deserializedWhatsAppMessageTemplateItem;
         });
     }
+
+    /*
+     * The template's name.
+     */
+    @Generated
+    private String name;
+
+    /**
+     * Get the name property: The template's name.
+     *
+     * @return the name value.
+     */
+    @Generated
+    @Override
+    public String getName() {
+        return this.name;
+    }
```

Comment:
@alzimmermsft for awareness.

This does not seem a big issue to me. Basically it's `shadowing` vs `Accessor method`, and both won't affect cause API level difference.